### PR TITLE
Add useTheme hook

### PR DESCRIPTION
**Pull request by karaxuna**
Change-type: minor

### CODE REVIEW

**Comment by karaxuna** [comment]
@balena-ci retest

**Comment by karaxuna** [comment]
@balena-ci retest

**Comment by karaxuna** [comment]
@balena-ci retest

File: __tests__/hooks/useTheme.spec.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+const Sample = () => {
+  const theme = useTheme()
+
+  return (
+    <span>{theme.foo}</span>
+  )
+}
+
+test.only('useTheme should return correct theme object', () => {
```
Comment (thgreasi) [inline __tests__/hooks/useTheme.spec.js]: `.only` :police_car: 
```suggestion
test.only('useTheme should return correct theme object', () => {
```

**Comment by karaxuna** [comment]
@balena-ci retest

**Comment by karaxuna** [comment]
@balena-ci retest

File: docs/_README.tpl.md [hunk 2]
```diff
@@ -51,6 +51,8 @@ For an interactive demo of all components, see https://balena-io-modules.github.
 
 Wrap your application in the `<Provider>` component so that child components can correctly inherit the default theme. You can optionally provide your own theme.
 
+You can access theme object from child components using `useTheme` hook or `withTheme` higher-order component.
```
Comment (LucianBuzzo) [inline docs/_README.tpl.md]: Can you include an example here? Also, could you provide a heading with the title `Accessing the theme object`.
```suggestion
You can the access theme object from child components using the `useTheme` hook or the `withTheme` higher-order component.
```

Comment (karaxuna) [reply docs/_README.tpl.md]: My grammar :))

There is an issue here. I can add `useTheme` example easily, but `withTheme` hoc is not exported from rendition, you have to import it from `styled-components`, so adding `withTheme` example would be wrong probably, until we export it from rendition.

Comment (LucianBuzzo) [reply docs/_README.tpl.md]: @karaxuna for now, lets remove the `withTheme` example until we have a better story for how to use it.

**Comment by karaxuna** [comment]
@balena-ci retest

**Comment by karaxuna** [comment]
@balena-ci retest

**Comment by karaxuna** [comment]
@balena-ci retest

**Comment by karaxuna** [comment]
@nazrhom I have not changed anything in form component, do you have an idea what could cause  different screenshot generation?

**Pull request merged**